mongo/bounties/final.json

```diff
@@ -0,0 +1,85 @@
+[
+  {
+    "_id": "60f6585b453e70eed340e8e3",
+    "season": 1,
+    "title": "Make new website landing page",
+    "description": "Description for new website landing page with specs",
+    "criteria": "A completed bounty has the following deliverables...",
+    "reward": {
+      "currency": "BANK",
+      "amount": 10000.00
+    },
+    "applicableGuilds": [
+      {
+        "guildName": "Developers Guild",
+        "id": "60f6585b453e70eed340e8e3"
+      },
+      {
+        "guildName": "Marketing Guild",
+        "id": "60f6585b453e70eed340e8e3"
+      }
+    ],
+    "createdBy": {
+      "_id": "60f6558d453e70eed340e8e0",
+      "isDaoMember": true,
+      "guildName": "Analytics Guild",
```

Comment:
Note: I think this should be an optional field. A bounty may lie outside of a guild, and a user may belong to many guilds. If the purpose of this field is to inform analytics, it may get polluted by bounties that aren't created for a guild.

Reply:
agree, just to be clear in final_validation.js only the object is required. So technically none of the fields within the object is required unless it's specified as

required: [
        "createdBy",
        "createdBy._id",
        "createdBy.guildName",
      ],
      
maybe we should specific some fields for the required object?